pkgs/servers/xmpp/biboumi/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, cmake, libuuid, expat, sqlite, git, libidn,
+  libiconv, botan2, systemd, pkgconfig, udns } :
+
+stdenv.mkDerivation rec {
+  name = "biboumi-${version}";
+  version = "6.1";
+
+  src = fetchurl {
+    url = "https://git.louiz.org/biboumi/snapshot/biboumi-${version}.tar.xz";
+    sha256 = "1la1n502v2wyfm0vl8v4m0hbigkkjchi21446n9mb203fz1cvr77";
+  };
+
+  buildInputs = [ cmake libuuid expat sqlite git libiconv libidn botan2
+    systemd pkgconfig udns ];
+
+  preConfigure = ''
+    grep -lr /etc/biboumi . | while read f
+    do
+      substituteInPlace $f --replace /etc/biboumi $out/etc/biboumi
```

Comment:
Does makes biboumi reading its configuration always from `$out` or can this be overwritten via command line?

Reply:
the config file is a cmdline argument, but "make install" fails without this adjustment.